cfme/services/workloads.py

```diff
@@ -41,41 +95,29 @@ def __init__(self, appliance=None):
 
 @navigator.register(VmsInstances, 'All')
 class AllVMs(CFMENavigateStep):
+    VIEW = WorkLoadsVM
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self, *args, **kwargs):
+        workloads = self.create_view(WorkloadsView)
```

Comment:
OPTIONAL: are you sure that's correct that you define one view in VIEW statement but create another one in step ?